delta/sdk/src/main/resources/contexts/metadata.json

```diff
@@ -17,11 +17,12 @@
       "@type": "@id"
     },
     "_instant": "https://bluebrain.github.io/nexus/vocabulary/instant",
-    "_subject": "https://bluebrain.github.io/nexus/vocabulary/subject",
+    "_subject": "https://bluebrain.github.io/nexus/vocabulary/metadata/subject",
```

Comment:
any reason for this change?

Reply:
Yes, because of collisions with the `subject` on ACLs and resolvers and others.

Reply:
I see. How did we handle this in <= v1.4?

Reply:
It was wrongly handled :)